app/src/main/java/umc/OnAirMate/ui/pop_up/CreateRoomPopup.kt

```
@@ -0,0 +1,143 @@
+            isPrivate = false
+        }
+        binding.ivPrivateRoomOff.setOnClickListener {
+            binding.ivPrivateRoomOn.visibility = View.VISIBLE
+            binding.ivPrivateRoomOff.visibility = View.GONE
+            isPrivate = true
+        }
+    }
+
+    private fun setTextListener(){
+        binding.etInputRoomTitle.addTextChangedListener(object : TextWatcher {
+            override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)